css/properties/justify-items.json

```diff
@@ -2,53 +2,108 @@
   "css": {
```

Comment:
Some of these files are changed because you based this PR off your previous PRs. That was OK since I was reviewing them sequentially, but since I'm suggesting some other changes, if you wouldn't mind rebasing or merging this (and your subsequent PRs, after you make changes), it'll make it easier to review these going forward. Thank you!

Reply:
I did merge between PRs, however that didn't seem to help. If I'm not doing lots I just start over each time by creating a new fork, and working from there. However that doesn't help with these small changes. I don't get on with GitHub at all!

Reply:
Yeah, GitHub PRs are awkward sometimes. I find that I look at this doc an awful lot: https://help.github.com/articles/syncing-a-fork/ If you follow those steps, you can update your fork's `master` from `upstream/master`, then merge `master` into your PR branches (or rebase your PR branches onto master). It's all a bit of a hassle though